src/_bentoml_impl/server/serving.py

```diff
@@ -301,6 +301,7 @@ def serve_http(
         )
 
         log_host = "localhost" if host in ["0.0.0.0", "::"] else host
+        dependency_map[svc.name] = f"{scheme}://{log_host}:{port}"
```

Comment:
why do we need this line again?

Reply:
To be consistent for entry services and dependent services. Users can do `EntryService.url` and won't get a null.